shared/constants/favorite.js

```diff
@@ -48,4 +55,41 @@ export type FavoriteSwitchTab = TypedAction<'favorite:favoriteSwitchTab', {showi
 export const favoriteToggleIgnored = 'favorite:favoriteToggleIgnored'
 export type FavoriteToggleIgnored = TypedAction<'favorite:favoriteToggleIgnored', {isPrivate: boolean}, void>
 
-export type FavoriteAction = FavoriteAdd | FavoriteList | FavoriteIgnore | FavoriteSwitchTab | FavoriteToggleIgnored
+export type FavoriteAction = FavoriteGet | FavoriteAdd | FavoriteList | FavoriteIgnore | FavoriteSwitchTab | FavoriteToggleIgnored
+
+function pathFromFolder ({isPublic, users}: {isPublic: boolean, users: UserList}) {
```

Comment:
just a heads up this has changed here: https://github.com/keybase/client/pull/4153/files#diff-71c2a1862130735603501d31c5580a36L16